charts/datadog/values.yaml

```diff
@@ -35,6 +35,7 @@ datadog:
   # datadog.apiKeyExistingSecret -- Use existing Secret which stores API key instead of creating a new one. The value should be set with the `api-key` key inside the secret.
 
   ## If set, this parameter takes precedence over "apiKey".
+  ## your DATADOG_API_KEY_SECRET secret should contain a key called 'api-key' containing the API key in base64 encoding with quotes.
```

Comment:
this information is already present line 35 https://github.com/DataDog/helm-charts/pull/948/files#diff-2fcec98016e714405bc936fd949012c2e72f23c20a1a94acf695a9cd3e29d75fL35 

we need to follow the format of ligne 35 so the comment is used to document the option in the chart [Readme.md](https://github.com/DataDog/helm-charts/tree/main/charts/datadog#readme)